licenses-report.txt

```diff
@@ -7,41 +7,154 @@ Apache-2.0
 ========================================================================
 The following software have components provided under the terms of this license:
 
+- ASM Analysis (from )
+- ASM Core (from )
+- ASM Tree (from )
+- ASM Util (from )
+- AWS Java SDK for AWS KMS (from https://aws.amazon.com/sdkforjava)
+- AWS Java SDK for AWS STS (from https://aws.amazon.com/sdkforjava)
+- AWS Java SDK for Amazon S3 (from https://aws.amazon.com/sdkforjava)
+- AWS SDK for Java - Core (from https://aws.amazon.com/sdkforjava)
 - Airline - IO Library (from )
 - Airline - Library (from )
 - An implementation of the Apache Cassandra® native protocol (from https://github.com/datastax/native-protocol)
 - Annotation Processing Commons (from https://github.com/misberner/ap-commons)
 - Annotations for DS (from )
+- Annotations for Metrics (from )
+- Apache Ant Core (from http://ant.apache.org/)
+- Apache Ant Launcher (from http://ant.apache.org/)
+- Apache Cassandra (from http://cassandra.apache.org)
+- Apache Commons BeanUtils (from http://commons.apache.org/proper/commons-beanutils/)
+- Apache Commons CLI (from http://commons.apache.org/proper/commons-cli/)
+- Apache Commons Codec (from http://commons.apache.org/proper/commons-codec/)
 - Apache Commons Collections (from http://commons.apache.org/proper/commons-collections/)
+- Apache Commons Compress (from http://commons.apache.org/proper/commons-compress/)
+- Apache Commons Configuration (from http://commons.apache.org/configuration/)
+- Apache Commons Exec (from http://commons.apache.org/proper/commons-exec/)
 - Apache Commons Lang (from http://commons.apache.org/proper/commons-lang/)
-- Apache Commons Lang (from http://commons.apache.org/proper/commons-lang/)
+- Apache Commons Logging (from http://commons.apache.org/proper/commons-logging/)
+- Apache Commons Math (from http://commons.apache.org/proper/commons-math/)
+- Apache Commons Text (from http://commons.apache.org/proper/commons-text/)
 - Apache Felix Declarative Services (from )
 - Apache Felix Framework (from )
+- Apache HttpClient (from http://hc.apache.org/httpcomponents-client)
+- Apache HttpCore (from http://hc.apache.org/httpcomponents-core-ga)
+- Apache MINA Core ${project.version} (from )
+- Apache Thrift (from http://thrift.apache.org)
+- Apache TinkerPop :: Gremlin Core (from )
+- Apache TinkerPop :: Gremlin Shaded (from )
+- Apache TinkerPop :: TinkerGraph Gremlin (from )
```

Comment:
`from` URL is missing (also in a few other places

Reply:
Yeah I guess fossa-cli wasn't able to figure it out?

Reply:
Do we want to have it in our report or is this meant to be 1:1 tool output?